### PR TITLE
Increased coverage of cosmos encryption

### PR DESCRIPTION
**Pull request by kushagraThapar**
# Description

Increased coverage of cosmos encryption

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by simplynaveen20 — APPROVED** [review]
LGTM , thanks @kushagraThapar

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/pom.xml [hunk 1]
```diff
@@ -14,6 +14,9 @@
       <id>coverage</id>
       <modules>
         <module>azure-cosmos</module>
+        <module>azure-spring-data-cosmos</module>
+        <module>azure-cosmos-spark_3_2-12</module>
+        <module>azure-cosmos-encryption</module>
```
Comment (alzimmermsft) [inline sdk/cosmos/pom.xml:19]: The modules will also need to be added as dependencies in the `dependencies` configuration below in the same way `azure-cosmos` was added. Jacoco needs to have the SDK as a dependency to generate aggregate coverage correctly.

**Review by alzimmermsft — APPROVED** [review]

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**